Gemfile.lock

```diff
@@ -108,12 +110,10 @@ GEM
     marcel (1.0.2)
     method_source (1.0.0)
     mini_mime (1.1.2)
-    mini_portile2 (2.8.0)
     minitest (5.15.0)
     msgpack (1.4.5)
     nio4r (2.5.8)
-    nokogiri (1.13.3)
-      mini_portile2 (~> 2.8.0)
+    nokogiri (1.13.3-x86_64-linux)
```

Comment:
Why are these changes generated?

Reply:
It should be generated automatically since I use the linux terminal when programming and compiling